app/components/primer/alpha/select_panel.html.erb

```diff
@@ -1,4 +1,16 @@
 <%= render Primer::BaseComponent.new(**@system_arguments) do %>
+  <% if acts_as_form_input? %>
+    <span data-select-panel-inputs="true"> 
+      <% if @default_values.present? && @default_values.length > 0 ? true : false  %>
+        <% for value in @default_values %>
+
+          <%= @form_builder.hidden_field(@input_name, multiple: multi_select?, skip_default_ids: true, value: value) %>
```

Comment:
Based on the current [multi-select form example](https://primer-lookbook.github.com/view-components/lookbook/inspect/primer/alpha/select_panel/multiselect_form) I think it's okay for all of these inputs to have the same name.